src/core/ext/transport/cronet/client/secure/cronet_channel_create.h

```diff
@@ -16,13 +16,14 @@
  *
  */
 
+#include <grpc/support/port_platform.h>
```

Comment:
These two includes should go inside the include-guard `#ifndef`.

Reply:
yikes,yeah